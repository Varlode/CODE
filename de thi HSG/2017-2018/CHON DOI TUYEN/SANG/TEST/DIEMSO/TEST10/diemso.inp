1000
811377 832374 68520 215189 122093 353697 276 311892 99026 68840 599931 990883 517892 762761 128462 631585 581665 18780 426300 573676 839945 233266 163033 651197 21827 83134 540130 962932 952692 406269 689298 978563 639876 104326 52085 496537 493317 282712 557623 14656 935719 809576 405067 789829 336284 53299 477417 192014 526546 309144 39770 99456 463970 417711 303118 699325 649628 575011 647734 69397 662361 457273 227448 210550 458477 804935 329239 808313 61263 999052 56843 585371 291621 960383 62529 265808 326832 761720 521406 922832 555698 676358 96813 982331 746001 220493 463090 77784 107805 448200 49151 220801 207418 901280 684915 996552 374525 609909 134076 760716 607380 457747 459264 582162 346763 486375 244146 521192 10219 969979 510396 971566 543290 98976 992156 140659 160165 270573 675341 156881 228979 389589 881744 423870 868488 364401 231843 171084 539562 795448 534652 462426 336395 25706 997903 277463 228764 228721 361510 603971 677499 696529 285226 742442 35749 954231 828786 660835 186731 903131 812388 765058 555656 346769 247823 537375 317572 524259 423871 261380 954633 142331 904138 736453 136080 298002 510077 667945 147201 670910 235571 627513 820408 9241 251541 983022 120164 885764 295244 98664 512184 989302 263970 449773 974099 976414 556719 385472 56821 931841 728537 863727 83229 293510 9064 368195 89211 627201 69605 276667 937932 917043 187886 393971 680394 161396 949721 344198 793335 779515 960593 668973 594141 865644 468095 876429 208578 804872 613968 849329 214139 805045 562484 764097 417260 663939 804984 362475 754672 277740 127768 952690 552448 430179 981750 778071 992113 370921 516309 235350 378862 397034 657809 702141 237446 37190 839725 781181 137990 757703 581534 294778 958789 241595 21750 147287 328877 271064 370862 597969 13213 10827 487430 679761 976089 205797 236186 790134 356532 404618 358954 443834 160225 486065 435126 411833 826858 779013 127966 50375 928144 207399 458134 66976 329249 658972 290417 573901 584971 668625 735076 248207 757245 579465 687132 161169 564787 264020 937179 784255 432718 17402 686628 474767 522187 811216 961811 921012 317510 857099 559115 66805 915038 720048 111309 493016 795792 977598 393447 699351 732464 828050 731579 882574 662064 202980 709640 731831 317135 441973 673629 166492 397909 483686 646994 603057 592197 332658 515042 1013 49302 142116 697874 477731 244154 294854 749149 861018 415617 323809 607224 70665 884217 965402 926603 898256 143276 973114 637483 619238 806734 745807 613807 489230 771879 955594 698274 839206 377506 533269 157603 261056 294948 806462 577510 148463 718041 485028 883504 499722 193897 802929 563394 601334 495487 994165 343420 476492 666256 774977 847197 283221 511132 754114 875717 320354 894181 356652 69490 471760 568704 786124 789129 847485 74624 271336 935226 182084 713496 956061 827222 568131 614909 548862 814379 770755 883295 741929 140924 828003 128747 528028 891904 33857 253162 787849 451855 759714 532107 192465 114181 91838 446884 83791 529592 115240 290945 67729 326169 934100 734117 646522 378654 666897 716597 762584 298566 47510 964105 536929 132719 511332 637400 107617 643111 730074 303329 492530 672403 134696 869155 33028 92914 997779 56533 835678 943731 612374 59702 660787 173178 69847 685208 697009 640799 911185 98978 419559 347807 163682 102959 632344 667661 604246 291930 916183 784939 264072 592146 672197 802837 298596 629654 157306 328764 90960 535016 436153 375286 949 479864 788830 233876 678202 415940 317761 258342 353428 229025 408932 851930 393830 423533 338642 915011 7967 434660 686250 158077 655851 978790 976288 321359 97522 313247 286999 760369 338636 979819 359048 945411 241613 975654 868091 684028 70458 150901 812461 218018 460331 258211 121838 387847 828928 973397 72370 879699 878802 396757 698408 565048 239166 482444 255451 389416 773470 560118 652697 783281 732439 723707 986174 187004 374593 115693 777932 891923 535385 17915 504355 275750 781194 732029 821663 674958 247102 485511 861876 695914 381941 762259 294190 627 408051 236896 586242 3120 715627 234503 664112 324554 679093 345885 868304 575836 6089 200707 779611 60464 995544 670395 306776 85864 199238 358939 217575 901947 937580 753389 983495 107191 471297 544190 378014 738613 493168 452682 31738 856301 767055 532910 27931 965785 214512 683536 752264 760412 331127 499201 512480 936407 35336 718236 952355 520260 635740 130206 372658 504164 312411 163815 468996 392897 521551 289588 935850 629281 686873 570683 938511 105590 661108 463868 668236 474100 962740 566026 729948 812451 324763 290655 272 262955 159324 825628 475998 987478 960887 455044 612799 965083 793895 419227 404428 678914 454017 207262 305210 569184 123961 308847 703935 428705 62829 482344 742433 362288 5969 904375 419405 392550 774969 533282 388539 700915 527056 356501 127701 692736 425560 705726 242965 521416 982398 16193 976461 823137 116681 662762 27890 428802 623762 270797 532683 557569 780424 271713 354246 14590 82976 101530 631819 174136 260479 405287 239083 793575 501968 500033 206380 95421 897619 665873 979625 309728 817286 53130 285645 42967 386814 510217 191577 599557 533020 594406 185390 997617 536974 883635 894184 717465 516993 322713 499799 380578 412363 169161 793881 667767 269163 37751 191640 459697 604122 262721 830069 676263 455362 523302 696816 596447 585694 542484 476499 979937 237896 925483 761337 763111 724890 790782 648586 772204 723787 823252 242735 965936 778045 76362 140671 598440 827005 404269 363614 498526 967676 608890 303867 694349 206740 699540 819505 714732 375222 394882 86389 133797 881159 386528 877936 881611 950910 684666 46051 418581 104873 575494 611813 336441 938215 508334 795837 367671 67300 588377 948805 130941 934039 44240 146274 504315 933706 151083 867144 589942 344624 833733 579673 153881 24110 355296 988778 740825 604587 941620 617980 227514 283100 362654 263011 751112 727216 811345 132668 979864 386514 238843 504218 324454 881845 660661 236868 224216 384700 540244 991344 115582 22080 690254 558707 315486 210849 509178 299503 207860 519908 395176 954265 914177 650756 890436 661720 419431 814100 788352 251177 641382 257208 853824 888637 413431 259681 809684 15858 832575 918718 950 485920 428039 577850 583421 703955 384867 99204 331793 64107 425010 946582 49963 558903 349838 503646 606304 166890 896356 488207 843636 978674 364446 767865 622469 548777 849129 394934 614513 519561 322609 901462 63062 444178 315556 856505 3341 223049 511703 809634 583995 836516 978041 767698 18438 52388 129378 690826 816432 657608 484357 146876 489825 470986 53741 841027 919141 900651 998147 441202 506121 29361 99694 692467 751850 80958 890592 216783 263845 434895 273021 698328 992854 125038 746298 952839 608352 468259 332581 769048 830836 490843 318135 345741 376373 600592 113461 989485 913537 122224 948133 21403 282493 454295 113948 107444 806617 322032 610800 449840 408271 656676 320266 52796 
3
37 186
55 188
84 165
