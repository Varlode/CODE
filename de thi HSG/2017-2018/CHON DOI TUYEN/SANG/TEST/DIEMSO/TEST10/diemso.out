71818198
65556279
39779741
